assets/templates/partials/release/contents/about-the-data.tmpl

```diff
@@ -1,11 +1,21 @@
 <div class="about-the-data">
   {{ if .Description.Census2021 }}
     <div class="ons-u-mb-l census-2021">
-      <img
-        src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg"
-        alt="Census 2021 logo"
-        class="ons-u-mb-xs"
-      >
+      {{ if eq .Language "en" }}    
+          <img
+            src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg"
+            alt="Census 2021 logo"
+            class="ons-u-mb-xs"
+            width="171"
+          >
+        {{ else }}
+          <img
+            src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-cyfrifiad-2021-purple-landscape.svg"
+            alt="Census 2021 logo"
+            class="ons-u-mb-xs"
+            width="171"
+          >
+        {{ end }}
```

Comment:
`if/else/end` indentation should be consistent.

Reply:
This done.